README.md

```diff
@@ -1,4 +1,6 @@
-Validoctor is an all-purpose data validator for backend java projects. It performs validation basing on rules passed along with object to be validated. It can operate depending on several traits and on various kinds of rules in an effort to cater to specific needs of all projects.
+Validoctor is an all-purpose data validator for backend java projects. It performs validation basing on rules passed 
+along with object to be validated. It can operate depending on several traits and on various kinds of rules in an effort 
+to cater to specific needs of all projects.
 
 # Getting started
```

Comment:
# Motivation Part   
Why you should use Validator instead of good old JSR 303 ?